docs/intro/installation.rst

```diff
@@ -4,84 +4,41 @@
 Installation
 ************
 
-The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and sunpy.
 
 Installing Python
 =================
-These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
+There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
+That's because we will create a new Python environment.
+As well as containing a Python installation, this environment provides an isolated place to install Python packages (like sunpy).
 
-To install the miniforge Python distribution follow `the miniforge installation instructions <https://github.com/conda-forge/miniforge#install>`__.
+The package manager we'll be using is called ``conda``.
+To install conda, follow the `the miniforge installation instructions <https://github.com/conda-forge/miniforge#install>`__.
+This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``, which is where ``sunpy`` packages are available.
 
-This makes installing ``sunpy`` and many other packages in the scientific Python ecosystem much easier and quicker.
-It also provides many pre-compiled binaries that are not available on PyPI.
+Installing sunpy
+----------------
+To install ``sunpy``, start by launching a terminal (under a UNIX-like system) or miniforge Prompt (under Windows).
+Then create a and activate new virtual environment
 
-Installing a package
---------------------
-To install ``sunpy``, launch a terminal (under a UNIX-like system) or miniforge Prompt (under Windows).
+.. code-block:: bash
 
-It is considered good practice to create a new virtual environment for each project you work on.
-`Here are some instructions on setting up a conda virtual environment. <https://towardsdatascience.com/getting-started-with-python-environments-using-conda-32e9f2779307>`__
+    $ conda create --name sunpy
+    $ conda activate sunpy
 
-You will want to activate your virtual environment then run:
+In this case the environment is named 'sunpy'.
+Feel free to change this to a different environment name.
+Now we have a fresh environment we can install sunpy:
 
 .. code-block:: bash
 
     $ conda install sunpy
```

Comment:
I dont know where exactly this should go, but what about cases where people already have their own conda installed (e.g. anaconda, miniconda) and want to install sunpy? should we have a sentance about conda-forge? I think we should. 

I realise this section is the tutorial of an introduction on how to do a fresh install of python and sunpy, but I imagine a lot of people will skip to the `Installing sunpy` section and follow the instructions of `conda install sunpy` and then it wont work without the conda-forge config or without doing `conda install -c conda-forge sunpy`.

what should we do in this case? maybe include a sentance here along the lines of 

"if you did not install python using miniforge you must first do 
```
conda config --add channels conda-forge
conda config --set channel_priority strict
``` 
on your new environment"

Reply:
Good idea - I think these lines can go below `conda activate sunpy` above. I assume if miniforge is installed they will just do nothing, but I'll check that.

Reply:
THey should do nothing but we should not suggest they do that if they have miniforge. 

Only if they have anaconda or miniconda only.

Reply:
👍 I added the channel commands